videos/tasks.py

```diff
@@ -101,26 +97,44 @@ def start_transcript_job(video_id: int):
         .first()
     )
 
-    if video_resource:
-        title = video_resource.title
-    else:
-        title = video.source_key.split("/")[-1]
+    title = video_resource.title
+    video_filename = video_resource.filename
+    captions = WebsiteContent.objects.filter(
+        Q(website=video.website) & Q(filename=f"{video_filename}_captions")
+    ).first()
+    transcript = WebsiteContent.objects.filter(
+        Q(website=video.website) & Q(filename=f"{video_filename}_transcript")
+    ).first()
+
+    if captions or transcript:  # check for existing captions or transcript
+        if captions:
+            video_resource.metadata["video_files"]["video_captions_file"] = str(
+                captions.file
+            )
+        if transcript:
+            video_resource.metadata["video_files"]["video_transcript_file"] = str(
```

Comment:
Also, these values are usually updated upon publish (via the `update_transcripts_for_video` task), with the s3 file path being replaced with the published website url path.  However, this is not happening because threeplay_api.update_transcripts_for_video(video) is returning `False`:

```def update_transcripts_for_video(video_id: int):
    """Update transcripts for a video"""
    video = Video.objects.get(id=video_id)
    if threeplay_api.update_transcripts_for_video(video):
        ....
        for video_resource in website.websitecontent_set.filter(**search_fields):
                metadata = video_resource.metadata
                set_dict_field(
                    metadata,
                    settings.YT_FIELD_TRANSCRIPT,
                    urljoin(
                        "/",
                        video.pdf_transcript_file.name.replace(
                            video.website.s3_path, video.website.url_path
                        ),
                    ),
                )
                set_dict_field(
                    metadata,
                    settings.YT_FIELD_CAPTIONS,
                    urljoin(
                        "/",
                        video.webvtt_transcript_file.name.replace(
                            video.website.s3_path, video.website.url_path
                        ),
                    ),
                )
```                
This function needs to be modified so that the above code is also run if tehre are pre-existing non-3play captions.